cfme/tests/containers/test_provider_crud.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+import pytest
+from utils import testgen
+from utils.providers import setup_a_provider as _setup_a_provider
+from utils.update import update
+from utils.version import current_version
+
+pytestmark = pytest.mark.uncollectif(lambda: current_version() < "5.6")
+
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP -9844 # CMP - 9846
+
+
+@pytest.yield_fixture(scope="function")
+def a_container_provider():
+    prov = _setup_a_provider("container")
```

Comment:
Yes, please close it.  I will take care about adding the CMPs to my test and update our sheet about them.
Thanks